From e262b6e7cc6271d71405f10c4817b9b3b2b95f05 Mon Sep 17 00:00:00 2001
From: Oleksandr Kravchuk <open.source@oleksandr-kravchuk.com>
Date: Mon, 13 Dec 2021 02:00:48 +0100
Subject: [PATCH] Include typeinfo for typeid()

Otherwise fails with:
    error: must '#include <typeinfo>' before using 'typeid'

Upstream-Status: Submitted
[https://github.com/dbus-cxx/dbus-cxx/pull/83]
---
 dbus-cxx/demangle.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/dbus-cxx/demangle.h b/dbus-cxx/demangle.h
index b71bcb9..9a4e99b 100644
--- a/dbus-cxx/demangle.h
+++ b/dbus-cxx/demangle.h
@@ -21,6 +21,7 @@
 #define DBUSCXX_DEMANGLE_H
 
 #include <string>
+#include <typeinfo>
 #include <dbus-cxx/dbus-cxx-config.h>
 
 #if DBUS_CXX_HAS_CXXABI_H
-- 
2.25.1

